nab/scorer.py

```diff
@@ -102,12 +102,12 @@ def scoreCorpus(threshold, args):
 
   # Using `map_async` instead of `map` so interrupts are properly handled.
   # See: http://stackoverflow.com/a/1408476
-  results = pool.map_async(scoreDataSet, args).get(99999999)
+  results = pool.map_async(scoreDataSet, args).get(999999)
```

Comment:
does it matter that this number changed?

Reply:
Yep there's an overflow error if this value is not reduced.

Reply:
can you make a comment why such value? I gues it's end of the list?

Reply:
Done.